integration/query_comparision_test.go

```diff
@@ -15,48 +15,235 @@
 package integration
 
 import (
-	"fmt"
+	"math"
 	"testing"
+	"time"
 
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
 	"go.mongodb.org/mongo-driver/mongo/options"
 
 	"github.com/FerretDB/FerretDB/integration/shareddata"
 )
 
-func TestQueryComparisionEq(t *testing.T) {
+func TestQueryComparisonEq(t *testing.T) {
```

Comment:
shall we have amount of tests to be passed also? 
then it would be clear how many tests were added / deleted. WDYT?

Reply:
All integration tests must pass. No need to track the count. What we should track is test coverage on codecov.io